build.gradle

```diff
@@ -289,6 +291,16 @@ subprojects {
         ])) {
             runtimeClasspath {
                 resolutionStrategy.failOnVersionConflict()
+
+                // We can remove this dependency override once org.apache.httpcomponents:httpclient:4.5.13
+                // is released with org.apache.httpcomponents:httpcore:4.4.14 dependency, and
+                // com.google.http-client:google-http-client:1.39.1 is upgraded with them.
+                // https://github.com/grpc/grpc-java/issues/8037
+                configurations.all {
+                    resolutionStrategy {
+                        force 'org.apache.httpcomponents:httpcore:4.4.14'
```

Comment:
I don't think this does what we need it to do. It may satisfy failOnVersionConflict(), but we have failOnVersionConflict() only to detect cases where Maven would interpret the dependencies incorrectly. Generally we have to exclude+redefine the dependency to force a specific transitive dependency version.

Reply:
Good. I'll update so.